process/pfcoil.py

```diff
@@ -964,13 +965,10 @@ def solv(self, ngrpmx, ngrp, nrws, gmat, bvec):
         :rtype: tuple[numpy.ndarray, numpy.ndarray, numpy.ndarray,
         numpy.ndarray, numpy.ndarray]
         """
-        truth = True
-        eps = 1.0e-10
         ccls = np.zeros(ngrpmx)
+        work2 = np.zeros(ngrpmx)
 
-        sigma, umat, vmat, ierr, work2 = ml.svd(
-            nrws, np.asfortranarray(gmat), truth, truth
-        )
+        umat, sigma, vmat = svd(gmat)
```

Comment:
The changes to PF coil require a bit of explaining. The single value decomposition from scipy returns different results to the maths library implementation. This causes the integration tests to fail, but none of the regression tests.

In some cases, like `test_solv` the values of `ccls` changed, but their sum did not. 
In `test_efc`, the `ccls` values changed (one of the PF coils groups ended up with 0 current!?) but the `ssq` value did not, weirdly, `ssq` is calculated from the `ccls`.

Overall, I suspect this is nothing to worry about, but its worth careful consideration before approving.